releasenotes/notes/add-cycle-basis-edges-5cb31eac7e41096d.yaml

```
@@ -0,0 +1,56 @@
+---
+features:
+  - |
+    A function, :func:`~rustworkx.cycle_basis_edges` was added to the crate
+    ``rustworkx-core`` in the ``connectivity`` module. This function returns 
+    the edge indices that form the cycle basis of a graph.
```

You need to split this into 2 release notes, one for the rustworkx-core function and one for the python side function being added.
 
```suggestion
  - |
    A function, ``cycle_basis_edges`` was added to the crate
    ``rustworkx-core`` in the ``connectivity`` module. This function returns 
    the edge indices that form the cycle basis of a graph.
  - |
    Added a new function :func:`~rustworkx.cycle_basis_edges`  which is similar
    to the existing :func:`~.cycle_basis` function but instead of returning node
    indices it returns a list of edge indices for the cycle.
```